examples/kmm-sample/build.gradle.kts

```diff
@@ -13,20 +12,10 @@ buildscript {
 group = "io.realm.example"
 version = Realm.version
 
-// Applying this here causes Gradle to hang, so applying in individual modules
-// subprojects {
-//    buildscript {
-//        repositories {
-//            maven(url = "http://oss.jfrog.org/artifactory/oss-snapshot-local")
-//        }
-//        dependencies {
-//            classpath("io.realm.kotlin:gradle-plugin:${Realm.version}")
-//        }
-//    }
-//    repositories {
-//        maven(url = "http://oss.jfrog.org/artifactory/oss-snapshot-local")
-//    }
-// }
-repositories {
-    mavenCentral()
+allprojects {
+    repositories {
+        gradlePluginPortal()
```

Comment:
Do we need to explicitly add `gradlePluginPortal` ?

Reply:
No, removed